settings.gradle.kts

```diff
@@ -1,5 +1,8 @@
-plugins {
-    id("org.gradle.toolchains.foojay-resolver-convention") version "0.7.0"
```

Comment:
I don't know what this is, but if it wasn't ever added purposefully, it's probably not needed in 2.x.